ui/job-view/headerbars/ActiveFilters.jsx

```diff
@@ -174,5 +174,5 @@ ActiveFilters.propTypes = {
   filterBarFilters: PropTypes.array.isRequired,
   isFieldFilterVisible: PropTypes.bool.isRequired,
   toggleFieldFilterVisible: PropTypes.func.isRequired,
-  classificationTypes: PropTypes.array.isRequired, // eslint-disable-line react/no-unused-prop-types
+  classificationTypes: PropTypes.array.isRequired,
```

Comment:
Another case that ESLint didn't handle properly before, which is why we had to add the `eslint-disable-line` even though it shouldn't have been necessary. Now that we're using the new `--report-unused-disable-directives` as part of `yarn lint`, this was flagged as an error since it's no longer required.